ydb/core/cms/console/console_handshake.cpp

```diff
@@ -0,0 +1,104 @@
+#include "defs.h"
+
+#include "configs_config.h"
+#include "console_impl.h"
+#include "console_configs_manager.h"
+#include <ydb/public/api/grpc/draft/ydb_dynamic_config_v1.grpc.pb.h>
+
+#include <ydb/core/base/tablet_pipe.h>
+
+#include <ydb/library/actors/core/hfunc.h>
+
+namespace NKikimr::NConsole {
+
+void TConfigsManager::Handle(TEvBlobStorage::TEvControllerProposeConfigRequest::TPtr &ev, const TActorContext &ctx) {
+    const auto& record = ev->Get()->Record;
+    const auto& proposedConfigHash = record.GetConfigHash();
+    const auto& proposedConfigVersion = record.GetConfigVersion();
+
+    auto currentConfigHash = std::hash<std::string>{}(YamlConfig);
```

Comment:
don't use std::hash -- it is not supposed to be stable
use xxhash, for example, it is the fastest for now